keps/349-all-or-nothing/README.md

```diff
@@ -315,9 +331,11 @@ type RequeueState struct {
 
 We introduce a new workload condition, called `PodsReady`, to indicate
 if the workload's startup requirements are satisfied. More precisely, we add
-the condition when `job.status.ready + job.status.succeeded` is greater or equal
+the condition when `job.status.ready + len(job.status.uncountedTerimnatedPods.succeeded) + job.status.succeeded` is greater or equal
```

Comment:
Does this indicate https://github.com/kubernetes-sigs/kueue/blob/6ac33dec0476a452c3406d39b15d9eaff74f44bc/pkg/controller/jobs/job/job_controller.go#L313-L316?

In that case, what if other jobs except batch/v1 Jobs?